### DISCUSSION TITLE
Show data from MicroPython using a website, similar to what blynk.io does

### DISCUSSION THREAD

**Discussion by mercurio20** [micropython/micropython#11069] [General]
Hi everybody,

Could you please recommend how to connect my device with MicroPython using a website, something similar to what blynk.io does
The problem with blynk.io is that I can not find enough info about how to use it with Micropython, apparently the support is kind of outdated.

I want to show the values from sensors and also control the outputs of my ESP8266 using Micropython using a website and/or Android App. You know show sensors' values and use buttons to activate/deactivate my ESP8266's outputs.

I'd appreciate some suggestions about how to do it, thanks

**Comment by kjm1102** [comment]
You can send data to the likes of ubidots etc but there are fees. If you're simply switching lights etc at home you can run mqqt to a mosquito server on your PC. Tasmota & ESPhome do this with 8266s but it's in C, not sure if the 8266 has enough grunt to do likewise in python? Google around a bit, if it's doable in python on an 8266 someone is bound to have done it & put the effort online.
